src/scripts/origins.js

```diff
@@ -637,5 +637,13 @@ export default {
     url: '*://app.vivifyscrum.com/*',
     name: 'VivifyScrum',
     clone: 'true'
+  },
+  'favro.com': {
```

Comment:
Hmm, it probably isn't necessary to have two entries here.

Just the first entry with `*://*.favro.com/*` as the `url` should work fine.

Reply:
Does that require the `.` before `favro` though? Don't want to accidentally match `myfavro.com` or other domains that end in the same string.